CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+### citus v10.0.3 (March 16, 2021) ###
+
+* Prevents infinite recursion for queries that involve UNION ALL below JOIN
+
+* Fixes a crash in queries with a modifying CTE and a SELECT without FROM
+
+* Fixes upgrade and downgrade paths for citus_update_table_statistics
+
+* Relaxes the locks when adding an existing node
+
+* Fixes a bug that causes SELECT queries to use 2PC unnecessarily
+
+* Skips 2PC for readonly connections in a transaction
+
+* Removes open temporary file warning when cancelling a query with
+  an open tuple store
+
+* Populates DATABASEOID cashe before CREATE INDEX CONCURRENTLY
```

Comment:
maybe:

Fixes a bug that might cause self-deadlocks with `CREATE INDEX / REINDEX CONCURRENTLY` commands

Reply:
it that makes sense, please also update the pr that brings that entry so that we can have consistent entries in different releases too